data/changelog.txt

```diff
@@ -147,438 +150,97 @@ Add reminder effects for some medicine.
-Adds new narcosis effect that Characters cannot be prematurely woken from
-Rolling papers made from one sheet of paper, not 5
-Add WATER_FRIENDLY flag to canteens, hip flasks, and waterskins
-Adds slowhealer2 and slowhealer3 traits
-Replaces heavy sticks with long sticks in many recipes
-50% chance of partial lighting in labs
-Kimonos are now fancy
-Lightstrips can be charged in the dark
-Adds solar backpack
-Recipes to extract seeds from some fruits and vegetables
-Extra requirements before an artifact will charged\
-Adds pickled eggs
-Additional doctor's office variant, a private bionics clinic
-CVD machine can now coat piercing weapons
-Bionic descriptions updated to match surgical installation
-Heavy duty variants of door, opaque door and trunk door
-Recipe for boy shorts
-Raw hides can be turned into simple bags to transport remains of creatures
-Adds variability in decay of food created before cataclysm
-Adds folding knife
-Rag bundles
-Add an electric clay kiln, portable and vehicle
-Jsonify military surplus
-Jsonify radio station
-Add Trencher (Construction vehicle)
-First draft of JSON-augmentation of labs
-Add chicken cages
-Add makeshift bandages to improvised first aid kit
-Jsonify pharmacy
-Blazemod: add magazines and increase clip capacities
-Lab JSON improvements
-Spawn dirt below the dumpsters of the fast food restaurant
-New Bow Attachments
-vehicle mounted pet carriers
-Updated railings in RetroDaysTileset
-Expanded Lab Sites
-Minor archery addition
+Acidic Ant Expansion - Acidic Chitin Item & Equipment.
+Add engine blocks and engine deconstruction recipes.
+Butchering yields for fungal towers and other structures.
+New basement variant, with and without hidden autodoc.
+Disposable filters for filter, gas, PBA, and survivor masks, as well as filters for hazmat and ANBC suits.
+Adds new narcosis effect that Characters cannot be prematurely woken from.
+Replaces heavy sticks with long sticks in many recipes.
+50% chance of partial lighting in labs.
+Recipes to extract seeds from some fruits and vegetables.
+Additional doctor's office variant, a private bionics clinic.
+Raw hides can be turned into simple bags to transport remains of creatures.
+Adds variability in decay of food created before cataclysm.
+Add Trencher (Construction vehicle).
+Vehicle mounted pet carriers.
 Discordant Mi-go Memes.
-Faction Camp
-Lab more floorplans
-Added sun glare protection for all kinds of survivor masks
-One more lab floorplan
-Add glare protection to all gear with glare quality
-To add Cosmic's Additional Locations to the game
+Add Cosmic's Additional Locations to the game.
 Root Cellar - food preserving option.
-Make leaf springs a thing
-Lab doors: Adds frosted glass, changes the mixture of doors in labs.
-Stacks of money
-Adds a debug trait that enables and gives 100% success chance on manual bionic installs
-Add special effects to 1 in 10 lab sections.
-Cosmic's Golf Course
-Mainline some of blaze and tank mods
```

Comment:
Added vehicle rams, since that was the most important change in most people's minds.

Reply:
Done.